agent/client/client.go

```diff
@@ -747,13 +747,6 @@ func dial(dialCtx context.Context, cfg *config.Config, l *logrus.Entry) (*dialRe
 		opts = append(opts, grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig)))
 	}
 
-	if cfg.Server.Username != "" {
-		opts = append(opts, grpc.WithPerRPCCredentials(&basicAuth{
-			username: cfg.Server.Username,
-			password: cfg.Server.Password,
-		}))
-	}
-
```

Comment:
let's keep it for now

Reply:
Is it really needed? I think 
https://github.com/percona/pmm/pull/2516/files#diff-e8867a3f753b0829c50d2b5e014024c244bc9081c1dd2b5e8b3d184ae666700cR91-R97
should do a same thing.

Reply:
that's for admin, this one for agent

Reply:
Sorry wrong link. Here is agent part which should do same I think:
https://github.com/percona/pmm/pull/2516/files#diff-1846b3b13f9217058effc538a7a76c52a7e4826ec9ba18179da9a47fa5a84596R106-R114

Reply:
and that's used only for `pmm-agent register` command, but not for `pmm-agent run`

Reply:
I see. Thanks for notice that.

Reply:
On the other hand, I'm not sure if it works at all 😊

Reply:
No problem. I will check it if it is difference.

Reply:
No difference, removed.